app/files/usr/local/etc/php/conf.d/docker-php-ext-zzz-dynamic.ini.ep

```diff
@@ -0,0 +1,5 @@
+[PHP]
+memory_limit=${PHP_MEMORY_LIMIT:-512M}
+post_max_size=${PHP_POST_MAX_SIZE:-32M}
+sendmail_path=${PHP_SENDMAIL_PATH:-/usr/sbin/sendmail -S host.docker.internal:1025 -t}
```

Comment:
The port in `host.docker.internal:1025` should probably be escaped with `\` or something. See the comment above.

Reply:
Ehm I don't understand. Why port should be escaped and which comment are you referring to?

Reply:
Aah, as `doas ep` read those envs and it fails on Linux I guess?

Reply:
I assume it fails because PHP_SENDMAIL_PATH is not defined and it fallbacks to `-/usr/sbin/sendmail -S host.docker.internal:1025 -t`, and it fails because it has `:` before port.